crates/iota-json-rpc-types/src/iota_transaction.rs

```diff
@@ -1157,7 +1157,12 @@ pub struct DryRunTransactionBlockResponse {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     #[schemars(with = "Option<BigInt<u64>>")]
     #[serde_as(as = "Option<BigInt<u64>>")]
-    pub suggested_gas_price: Option<u64>,
+    pub suggested_gas_price_with_ogd: Option<u64>,
```

Comment:
See my previous comment about keeping `suggested_gas_price` for Sui's prediction